reaper/client.go

```diff
@@ -33,303 +27,85 @@ type ReaperClient interface {
 
 	DeleteCluster(ctx context.Context, cluster string) error
 
-	RepairSchedules(ctx context.Context) ([]RepairSchedule, error)
+	// GetRepairRuns returns a list of repair runs, optionally filtering according to the provided search options.
+	GetRepairRuns(ctx context.Context, searchOptions *RepairRunSearchOptions) (map[uuid.UUID]*RepairRun, error)
 
-	RepairSchedulesForCluster(ctx context.Context, clusterName string) ([]RepairSchedule, error)
-}
+	// GetRepairRun returns a repair run object identified by its id.
+	GetRepairRun(ctx context.Context, repairRunId uuid.UUID) (*RepairRun, error)
 
-type Client struct {
-	BaseURL    *url.URL
-	UserAgent  string
-	httpClient *http.Client
-}
+	// CreateRepairRun creates a new repair run for the given cluster and keyspace. Does not actually trigger the run:
+	// creating a repair run includes generating the repair segments. Returns the id of the newly-created repair run if
+	// successful. The owner name can be any string identifying the owner.
+	CreateRepairRun(
```

Comment:
Maybe this is a question for @adejanovski. When I create a RepairRun, do I typically want to start it?

Reply:
It depends 😅 
So, I'd say not necessarily instantly if you first want to check how many segments you end up with.